packages/commons/README.md

```diff
@@ -21,6 +21,30 @@ This package exists primarily to help .NET tool developers integrate `axe-core`
 * [Deque.AxeCore.Playwright](../playwright/README.md) in combination with [Playwright for .NET](https://playwright.dev/dotnet/)
 * [Deque.AxeCore.Selenium](../selenium/README.md) in combination with [Selenium](https://www.selenium.dev/)'s [C# Selenium.WebDriver package](https://www.nuget.org/packages/Selenium.WebDriver)
 
+
+## `CachedContentDownloader`
```

Comment:
`ContentDownloader` and `CachedContentDownloader` are both internal implementation details of `ExternalAxeScriptProvider` - they shouldn't be documented since they are (intentionally) not public

Reply:
Removed